ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -36,6 +36,11 @@ set SDCARD_FORMAT no
 set STARTUP_TUNE 1
 set VEHICLE_TYPE none
 
+# Airframe parameter versioning: airframe maintainers can optionally set this in the
+# airframe startup script, and then increase it by one whenever they want to ensure
+# that parameters are reset to the airframe defaults for an update.
+set PARAM_DEFAULTS_VER 1
```

Comment:
```suggestion
# Set to 1 by default, can get overwritten in the airframe startup script (config file).
set PARAM_DEFAULTS_VER 1
```

Reply:
I wrote it slightly differently to not contain redundant statements but the points you suggested to add. See d7437b4